CHANGELOG.rst

```diff
@@ -5,13 +5,35 @@ Cisco ACI Ansible Collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Release Summary
+---------------
+
+Release v2.0.0 of the ``cisco.aci`` collection on 2020-12-15.
+This changelog describes all changes made to the modules and plugins included in this collection since v1.1.1. 
+
+Major Changes
+-------------
+
+- Change certificate_name to name in aci_aaa_user_certificate module for query operation
+
+Minor Changes
+-------------
+
+- Add aci_node_mgmt_epg module to manage in band or out of band management EPGs
+- Add aci_static_node_inb_mgmt_address module & test file
```

Comment:
this line is incorrect, the next line is the correct version we renamed the module.